.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,72 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/dotnet
+{
+	"name": "C# (.NET)",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/dotnet:1-9.0",
+	"features": {
+		"ghcr.io/devcontainers/features/azure-cli:1": {},
+		"ghcr.io/azure/azure-dev/azd:0": {},
```

Comment:
Useful for testing deployments.

Reply:
What version of azd is this?

Reply:
Don't know but we might be able to override it.

Reply:
@ellismg @vhvb1989 we are looking at enabling dev containers / codespaces in the dotnet/aspire repo. We are preinstalling azd and it looks like I can pin to a specific version as long as it has been uploaded to the GitHub releases page for azd, but is there a way to make the container image install a version from a PR too?

Reply:
@davidfowl to answer your earlier question this is what I found in my spelunking:

https://github.com/Azure/azure-dev/blob/b15ccaf35f60e30095740123f6beaa14c3c0f0fb/ext/devcontainer/src/azd/README.md

Reply:
Using the azd feature to pre-install will give yo the latest version at the momento of building the codespace.
The feature does not support installing versions from PRs, but you can just run the azd-install script from the codespace to change whatever version.
Note: if you ever want to use a [prebuilt codespace](https://docs.github.com/en/codespaces/prebuilding-your-codespaces) you might want to change from using the azd-feature to pre-install azd for using a postCreate action in your devcontainer to run azd's install script instead.  I mention this b/c prebuilt codespaces are getting more and more popular (for how fast you load your workspace) but keeping dependencies from features up to date becomes a little messy.